sdk/storage/Azure.Storage.Blobs/src/AppendBlobClient.cs

```diff
@@ -330,6 +330,175 @@ protected sealed override BlobBaseClient WithCustomerProvidedKeyImpl(CustomerPro
                 cancellationToken)
                 .ConfigureAwait(false);
 
+        /// <summary>
+        /// The <see cref="CreateIfNotExists"/> operation creates a new 0-length
+        /// append blob.  If the append blob already exists, the content of
+        /// the existing append blob will remain unchanged.  To add content to the append
+        /// blob, call the <see cref="AppendBlockAsync"/> operation.
```

Comment:
This should explain how the caller can know whether the blob was created or not.

Reply:
That can happen as part of #7896.